Production/govuk_ios/Views/RecentlyViewed/RecentActivityErrorView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
```

Comment:
You should be importing `Foundation` I think here. Rarely is there a case you wont be using `Foundation`